packages/mdc-drawer/README.md

```diff
@@ -301,15 +351,29 @@ Method Signature | Description
 `focusActiveNavigationItem() => void` | Focuses the active / selected navigation item.
 `notifyClose() => void` | Emits the `MDCDrawer:close` event.
 `notifyOpen() => void` | Emits the `MDCDrawer:open` event.
+`trapFocus() => void` | Traps focus on root element and focuses the active navigation element.
+`releaseFocus() => void` | Releases focus on root element which trapFocus was previously called on and resumes focus to nav button.
```

Comment:
This technically restores focus to whatever had it before, right? Maybe we should loosen the wording?

"Releases focus from the root element and restores it to where it was prior to calling `trapFocus`."

(Also update adapter doc comment)